build/media_source/com_content/js/admin-article-readmore.es6.js

```diff
@@ -20,8 +20,9 @@
     } else if (content && !content.match(/<hr\s+id=("|')system-readmore("|')\s*\/*>/i)) {
       Joomla.editors.instances[editor].replaceSelection('<hr id="system-readmore">');
     } else {
-      // @todo replace with joomla-alert
-      alert(options.exists);
+      Joomla.renderMessages({
+        error: [options.exists]
```

Comment:
```suggestion
        error: [options.exists],
```
Fix javascript code style error reported by drone here: https://ci.joomla.org/joomla/joomla-cms/60552/1/26